extensions/mssql/src/hdfs/ui/hdfsManageAccessDialog.ts

```diff
@@ -130,7 +130,18 @@ export class ManageAccessDialog {
 				rootContainer.addItem(typeContainer, { flex: '0 0 auto' });
 				const addUserOrGroupInputRow = modelView.modelBuilder.flexContainer().withLayout({ flexFlow: 'row' }).component();
 
-				this.addUserOrGroupInput = modelView.modelBuilder.inputBox().withProperties<azdata.InputBoxProperties>({ inputType: 'text', placeHolder: enterNamePlaceholder, width: 250 }).component();
+				this.addUserOrGroupInput = modelView.modelBuilder.inputBox()
+					.withProperties<azdata.InputBoxProperties>(
+						{
+							inputType: 'text',
+							placeHolder: enterNamePlaceholder,
+							width: 250,
+							stopEnterPropagation: true
+						}).component();
+				this.addUserOrGroupInput.onInputEntered((value: string) => {
```

Comment:
IIRC, you are planning to do disposable handling later, right?

Reply:
Correct. That turned out to be more complicated than I thought so probably won't get to it until after the Manage Access dialog stuff is finished though.